nh_eobs_mental_health/models/nh_clinical_wardboard.py

```diff
@@ -297,7 +297,13 @@ def read(self, cr, user, ids, fields=None, context=None,
                     ],
                     context=context)
                 rec['rapid_tranq'] = spell.get('rapid_tranq')
-                rec['next_blood_glucose_diff'] = rec['next_blood_glucose_diff'] if rec['next_blood_glucose_diff'] != '00:00' else ''
+
+                if rec.get('next_blood_glucose_diff'):
+                    rec['next_blood_glucose_diff'] = rec['next_blood_glucose_diff'] \
+                        if rec['next_blood_glucose_diff'] != '00:00' else ''
+                else:
+                    rec['next_blood_glucose_diff'] = ''
```

Comment:
This can probably be done better by inverting the check like:
```python
if rec.get('next_blood_glucose_diff') and rec.get('next_blood_glucose_diff') == '00:00':
    rec['next_blood_glucose_diff'] = ''
```
as I can't see any feasable reason to set `next_blood_glucose_diff` to itself